pages/algorithms/exposed-to-assistive-technologies.md

```diff
@@ -1,5 +1,11 @@
 ---
 title: Exposed to assistive technologies
+key: exposed-to-assistive-technologies
 ---
 
-Elements should be contained in the accessibility tree as described in [Core Accessibility API Mappings 1.1](https://www.w3.org/TR/core-aam-1.1/#mapping_general)
+Elements contained in the accessibility tree as described in one of the following documents:
+
+- [Core Accessibility API Mappings 1.1](https://www.w3.org/TR/core-aam-1.1/#mapping_general) 
```

Comment:
Should we make this unversioned? E.g. https://www.w3.org/TR/core-aam/#mapping_general